tools/sda-admin

```diff
@@ -0,0 +1,959 @@
+#!/bin/sh
+
+set -u -e
+
+myself=$0
+
+# Default values for global options.  Values in the environment override
+# these hard-coded defaults, and values set via command line options
+# override these later.
+
+MQ_CREDENTIALS=${MQ_CREDENTIALS-test:test}		# --mq-credentials user:pass
+MQ_URL=${MQ_URL-http://localhost:15672}			# --mq-url URL
+MQ_EXCHANGE=${MQ_EXCHANGE-sda}		# --mq-exchange string
+MQ_VHOST=${MQ_VHOST-sda}			# --mq-vhost string
+
+SDA_CONFIG=${SDA_CONFIG-s3cmd.conf}			# --sda-config pathname
+SDA_KEY=${SDA_KEY-crypt4gh_key.pub}			# --sda-key pathname
+
```

Comment:
Take the opportunity to change these to more sane names, for example:
* SDA_CONFIG (--sda-config) --> S3_config (--s3-config)
* SDA_KEY (--sda-key) --> C4GH_PUB_KEY (--c4gh-pub-key) or SUBMISSION_PUB_KEY

Names of flags and ENVs should be clear what they are and where they come from.